src/cellTypes/autocompleteType/autocompleteType.js

```diff
@@ -0,0 +1,10 @@
+import { AutocompleteEditor } from '../../editors/autocompleteEditor';
+import { autocompleteRenderer } from '../../renderers/autocompleteRenderer';
+import { autocompleteValidator } from '../../validators/autocompleteValidator';
+
+export const CELL_TYPE = 'autocomplete';
+export const AutocompleteType = {
```

Comment:
As the modules will now be part of the public API, and renaming the named exports could involve the breaking change. Maybe it's a better idea to rename the cell types to their full name like:  `AutocompleteCellType`? (the same I'd include to all cell types).